packages/grafana-ui/src/components/uPlot/config.ts

```diff
@@ -1,4 +1,5 @@
 import { SelectableValue } from '@grafana/data';
+import { ScaleDistribution } from './models.gen';
```

Comment:
it looks like many/most of these are duplicated in `models.gen` -- we can make another PR to move the core types out of uPlot and into the options folder